### PR TITLE
Data Prepper - Getting Started - updates 

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
Fixes issue 1823 and minor tweaks

Worked with David V. to confirm change to the Docker instructions.

### Issues Resolved
Fixes https://github.com/opensearch-project/documentation-website/issues/1823

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _data-prepper/getting-started.md [hunk 1]
```diff
@@ -10,40 +10,32 @@ redirect_from:
...
-There are two ways to install Data Prepper:
+There are two ways to install Data Prepper: Run the Docker image and build from source.
 
-1. Run the Docker image.
-2. Build from source.
-
-The easiest way to use Data Prepper is by running the Docker image. We suggest that you use this approach if you have [Docker](https://www.docker.com) available.
-
-You can pull the Docker image:
+The easiest way to use Data Prepper is by running the Docker image. We suggest that you use this approach if you have [Docker](https://www.docker.com) available. Do the following:
```
Comment (carolxob) [inline _data-prepper/getting-started.md:20]: Can we assume that users will know to enter this command in a terminal?

File: _data-prepper/getting-started.md [hunk 2]
```diff
@@ -10,40 +10,32 @@ redirect_from:
...
-You will configure two files:
+You must configure Data Prepper with a pipeline before running it. You'll modify the following files:
 
 * `data-prepper-config.yaml`
 * `pipelines.yaml`
 
-Depending on your use case, we have a few different guides to configuring Data Prepper.
+To configure Data Prepper, see the following information for each use case: 
 
 * [Trace Analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/): Learn how to collect trace data and customize a pipeline that ingests and transforms that data.
```
Comment (carolxob) [inline _data-prepper/getting-started.md:37]:
```suggestion
* [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/): Learn how to collect trace data and customize a pipeline that ingests and transforms that data. 
```

File: _data-prepper/getting-started.md [hunk 3]
```diff
@@ -10,40 +10,32 @@ redirect_from:
...
+You must configure Data Prepper with a pipeline before running it. You'll modify the following files:
 
 * `data-prepper-config.yaml`
 * `pipelines.yaml`
 
-Depending on your use case, we have a few different guides to configuring Data Prepper.
+To configure Data Prepper, see the following information for each use case: 
 
 * [Trace Analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/): Learn how to collect trace data and customize a pipeline that ingests and transforms that data. 
 * [Log Analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/log-analytics/): Learn how to set up Data Prepper for log observability.
```
Comment (carolxob) [inline _data-prepper/getting-started.md:38]:
```suggestion
* [Log analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/log-analytics/): Learn how to set up Data Prepper for log observability.
```

File: _data-prepper/getting-started.md [hunk 4]
```diff
@@ -65,11 +57,12 @@ Run the following command with your pipeline configuration YAML.
 
 ```bash
 docker run --name data-prepper \
-    -v /full/path/to/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml \
+    -v /${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml \
     opensearchproject/data-prepper:latest
+    
 ```
```
Comment (carolxob) [inline _data-prepper/getting-started.md:64]: I have added copy buttons to Data Prepper docs that ask you to create YAML files like this, and think this section would benefit from it.

File: _data-prepper/getting-started.md [hunk 5]
```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
...
-However you configure your pipeline, you will run Data Prepper the same way. You run the Docker
-image and supply both the `pipelines.yaml` and `data-prepper-config.yaml` files.
+However you configure your pipeline, you'll run Data Prepper the same way. You run the Docker
+image and modify both the `pipelines.yaml` and `data-prepper-config.yaml` files.
 
 For Data Prepper 2.0 or later, use this command:
 
 ```
 docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
 ```
```
Comment (carolxob) [inline _data-prepper/getting-started.md:95]: Add `{% include copy.html %}`.

File: _data-prepper/getting-started.md [hunk 6]
```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
...
 
-For Data Prepper before version 2.0, use this command:
+For Data Prepper 2.0 and earlier, use this command:
 
 ```
 docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:1.x
 ```
```
Comment (carolxob) [inline _data-prepper/getting-started.md:102]: Add `{% include copy.html %}`.

File: _data-prepper/getting-started.md [hunk 7]
```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
...
 
-Once Data Prepper is running, it will process data until it is shut down. Once you are done, shut it down with the following command:
+Once Data Prepper is running, it processes data until it is shut down. Once you are done, shut it down with the following command:
 
 ```
 curl -X POST http://localhost:4900/shutdown
 ```
```
Comment (carolxob) [inline _data-prepper/getting-started.md:107]: Add `{% include copy.html %}`.

File: _data-prepper/getting-started.md [hunk 8]
```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
...
 
-For Data Prepper 1.5 or earlier, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would 
-like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper will default to the log4j2.properties file in the *shared-config* directory.
+For Data Prepper 1.5 or earlier, optionally add `"-Dlog4j.configurationFile=config/log4j2.properties"` to the command if you would like to pass a custom log4j2 properties file. If no properties file is provided, Data Prepper defaults to the log4j2.properties file in the *shared-config* directory.
 
 ## Next steps
 
-Trace Analytics is an important Data Prepper use case. If you haven't yet configured it, see the [Trace Analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/).
+Trace analytics is an important Data Prepper use case. If you haven't yet configured it, see [Trace Analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/).
```
Comment (carolxob) [inline _data-prepper/getting-started.md:117]:
```suggestion
Trace analytics is an important Data Prepper use case. If you haven't yet configured it, see [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/).
```

**Review by carolxob — APPROVED** [review]
Looks good with some suggestions, mainly the addition of copy buttons to make it easier to use commands.

**Review by ariamarble — APPROVED** [review]
Looks good. I don’t have anything to add besides what’s already been commented.

File: _data-prepper/getting-started.md (hunk 4 above)
Comment (hdhalter) [reply _data-prepper/getting-started.md:64]: I was waiting for someone would point that out. :)  Do I need this command? {% include copy-curl.html %}

File: _data-prepper/getting-started.md (hunk 1 above)
Comment (hdhalter) [reply _data-prepper/getting-started.md:20]: We don't typically point that out in our other docs, but happy to be proven wrong.

File: _data-prepper/getting-started.md [hunk 9]
```diff
@@ -90,40 +83,38 @@ e51e700e-5cab-4f6d-879a-1c3235a77d18
...
+image and modify both the `pipelines.yaml` and `data-prepper-config.yaml` files.
 
 For Data Prepper 2.0 or later, use this command:
 
 ```
 docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
 ```
 
-For Data Prepper before version 2.0, use this command:
+For Data Prepper 2.0 and earlier, use this command:
```
Comment (JeffH-AWS) [inline _data-prepper/getting-started.md:97]: This changes the meaning of the sentence.

Old: Version < 2.0
New: Version ≤ 2.0

Recommend changing to "For Data Prepper versions earlier than 2.0..." or something similar.

Comment (JeffH-AWS) [reply _data-prepper/getting-started.md:97]: `opensearchproject/data-prepper:1.x` won't pull 2.0 so I think we should definitely fix that.

**Pull request merged**